services/docker-compose.yml

```diff
@@ -29,8 +29,8 @@ services:
       - POSTGRES_USER=${POSTGRES_USER}
       - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
       - POSTGRES_DB=${POSTGRES_DB}
-      - POSTGRES_HOST=${POSTGRES_HOST}
-      - POSTGRES_PORT=${POSTGRES_PORT}
+      - POSTGRES_HOST=postgres
+      - POSTGRES_PORT=5432
```

Comment:
why do you set it hard-coded?

Reply:
well with postgres there is also the POSTGRES_ENDPOINT. I think we shall keep either the endpoint or this host+port. This is confusing as I thought the endpoint was to be removed.
Now concerning the location of postgres, I think you need to check with @mguidon whether it needs to be separated or not. If that is the case, maybe we should have a separate stack.